### ISSUE TITLE
Add button role to improve accessibility in common/auth

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4110]
# Description:
Standardizing the appearance of buttons enhances the user's recognition of the widgets as buttons. We can add buttons by using `role=button` in the HTML element and thus can increase the accessibility of platform.

# Todo:
Go through the files in this folder: [platform-client/app/common/auth](https://github.com/ushahidi/platform-client/tree/develop/app/common/auth)

Search for any buttons in the codebase of platform-client which are not button-elements and add `role=button` in the html element with that class. An easy way to search that might be by searching for `class="button` Make sure that you do not make any changes to the UI of platform.

**Comment by Somaru-chan** [comment]
Hi can I work on this?

**Comment by renujain31** [comment]
Hey! Ya sure @Somaru-chan Thanks for contributing!

**Comment by Somaru-chan** [comment]
> Hey! Ya sure @Somaru-chan Thanks for contributing!

Hey @renujain31 . Thanks for assigning this issue to me. Are you sure there are non-button elements in the /auth directory though? I've looked through the files and the only matches I've found to "class="button were button elements. :)

Such as in demo-deployment.html:
`<button class="button" ui-sref="settings.plan" translate="message.button.upgrade">`
